graspy/inference/latent_distribution_test.py

```diff
@@ -14,121 +14,207 @@
 
 import numpy as np
 
-from ..embed import AdjacencySpectralEmbed, select_dimension
+from ..embed import select_dimension, AdjacencySpectralEmbed
 from ..utils import import_graph, is_symmetric
 from .base import BaseInference
+from sklearn.metrics import pairwise_distances
+from hyppo.ksample import KSample
+from scipy.stats import multiscale_graphcorr
+from joblib import Parallel, delayed
 
 
 class LatentDistributionTest(BaseInference):
     """
-    Two-sample hypothesis test for the problem of determining whether two random 
+    Two-sample hypothesis test for the problem of determining whether two random
     dot product graphs have the same distributions of latent positions.
-    
+
     This test can operate on two graphs where there is no known matching between
-    the vertices of the two graphs, or even when the number of vertices is different. 
+    the vertices of the two graphs, or even when the number of vertices is different.
     Currently, testing is only supported for undirected graphs.
 
     Read more in the :ref:`tutorials <inference_tutorials>`
 
     Parameters
     ----------
+    test : str
+        Independence test to use, one of ["CCA", "Dcorr", "HHG", "RV", "Hsic", "MGC"].
```

Comment:
update

Reply:
what's the issue here? that they should be lower-case?

Reply:
@bdpedigo changed to lower case.

Reply:
thanks